### PR TITLE
DPS-96: Add Dockerfile

### PR DESCRIPTION
**Pull request by hariharan-oviva**
This image will be pulled by the actions in the config repos. We use multi stage build to reduce the size of the image.

### CODE REVIEW

**Comment by continuousintegration-oviva** [comment]
[🔗 Jira Issue](https://oviva-ag.atlassian.net/browse/DPS-96)

**Review by thomasrichner-oviva — DISMISSED** [review]
👍 

In case you are wondering if you can go smaller: https://chemidy.medium.com/create-the-smallest-and-secured-golang-docker-image-based-on-scratch-4752223b7324

**Review by demianjaeger-oviva — DISMISSED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM golang:1.15-alpine AS build
+LABEL org.opencontainers.image.source https://github.com/oviva-ag/apnvalidator
+
+WORKDIR /src/
+COPY . /src/
```
Comment (thomasrichner-oviva) [inline Dockerfile]:
```suggestion
COPY . .
```

**Review by thomasrichner-oviva — DISMISSED** [review]
Nice! 👍

**Comment by hariharan-oviva** [comment]
Sorry, the reviews keep getting dismissed after every little change 😆

**Review by demianjaeger-oviva — APPROVED** [review]

**Pull request merged**